### PR TITLE
#76 Webpage Review 2

### PR DESCRIPTION
**Pull request by juliavisserinbibo**
## Issue
#76 

## Description
This PR addresses all the small changes the team found that must be made to the website before the release of version 1.
You can find the changes done in the Issue description

### Note
I tried to divide the commits by the reports done by each team member.

### CODE REVIEW

File: docs/integration_resources/software/houdini.md [hunk 1]
```diff
@@ -4,9 +4,12 @@
...
 
-<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">import sys
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt")
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt/shift/thirdparty/python/Lib/site-packages")
+<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">try:
+    import shift
+exception:
+    import sys
+    sys.path.append("<path_to_the_shift_installation_folder>")
+    sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
```
Comment (JorgeHernandezibanezinbibo) [inline docs/integration_resources/software/houdini.md:12]: This section does not display nicely
image
May be related to the < > or other thing

File: docs/integration_resources/software/maya.md [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
 
-<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">import sys
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt")
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt/shift/thirdparty/python/Lib/site-packages")
+<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">try:
+    import shift
+exception:
+    import sys
+    sys.path.append("<path_to_the_shift_installation_folder>")
+    sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
```
Comment (JorgeHernandezibanezinbibo) [inline docs/integration_resources/software/maya.md:12]: Same problem
image

File: docs/integration_resources/software/resolve.md [hunk 3]
```diff
@@ -48,9 +48,12 @@ For a Windows local user, the result path for the file will be this one:
...
 
-<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">import sys
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt")
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt/shift/thirdparty/python/Lib/site-packages")
+<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">try:
+    import shift
+exception:
+    import sys
+    sys.path.append("<path_to_the_shift_installation_folder>")
+    sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
```
Comment (JorgeHernandezibanezinbibo) [inline docs/integration_resources/software/resolve.md:56]: Same problem
image

File: docs/release_notes.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+- USD resources:
+    - In-build catalogs: sUsd, sUsdTypes, sMayaUsd.
+    - In-build plugins: Usd Outliner, Usd Viewer.
+
+
+### Known Limitations
+
+#### Integrations & Resources
+
+- USD Viewer does not currently work in Houdini.
```
Comment (JorgeHernandezibanezinbibo) [inline docs/release_notes.md:36]: After the test from @leegeertseninbibo maybe we have to remove this limitation

File: docs/release_notes.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
+    - Houdini: *hshift.py*.
+    - Nuke: *nshift.py*.
+- USD resources:
+    - In-build catalogs: sUsd, sUsdTypes, sMayaUsd.
+    - In-build plugins: Usd Outliner, Usd Viewer.
+
+
+### Known Limitations
+
+#### Integrations & Resources
```
Comment (JorgeHernandezibanezinbibo) [inline docs/release_notes.md:34]: Duplicated header

File: docs/release_notes.md [hunk 6]
```diff
@@ -1,5 +1,36 @@
...
+
+- Scripts to bind Shift interface to a DCC for:
+    - Maya: *mshift.py*.
+    - Houdini: *hshift.py*.
+    - Nuke: *nshift.py*.
+- USD resources:
+    - In-build catalogs: sUsd, sUsdTypes, sMayaUsd.
+    - In-build plugins: Usd Outliner, Usd Viewer.
+
```
Comment (JorgeHernandezibanezinbibo) [inline docs/release_notes.md:30]: Should we list the Examples like Resources here? @Inbibo/shift

**Review by JorgeHernandezibanezinbibo — CHANGES_REQUESTED** [review]

File: docs/reference/developer_guide/api.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
 # Shift API
 
 The developers documentation can be found at the following location in the installation folder:
-`docs/html/index.html`
+`<path_to_the_shift_installation_folder>/docs/html/index.html`
```
Comment (leegeertseninbibo) [inline docs/reference/developer_guide/api.md:4]:
```suggestion
`&ltpath_to_the_shift_installation_folder&gt/docs/html/index.html`
```

**Review by leegeertseninbibo — CHANGES_REQUESTED** [review]

File: docs/reference/developer_guide/api.md (hunk 7 above)
Comment (juliavisserinbibo) [reply docs/reference/developer_guide/api.md:4]: This is showing up correctly
image

File: docs/integration_resources/software/maya.md (hunk 2 above)
Comment (juliavisserinbibo) [reply docs/integration_resources/software/maya.md]: Done
image

File: docs/integration_resources/software/resolve.md (hunk 3 above)
Comment (juliavisserinbibo) [reply docs/integration_resources/software/resolve.md]: Done

File: docs/integration_resources/software/houdini.md (hunk 1 above)
Comment (juliavisserinbibo) [reply docs/integration_resources/software/houdini.md]: Done

File: docs/release_notes.md (hunk 5 above)
Comment (juliavisserinbibo) [reply docs/release_notes.md:34]: This header is under the "Known Limitations" section.
The other header is under "Added" section.

File: docs/release_notes.md (hunk 6 above)
Comment (hectorjimenezinbibo) [reply docs/release_notes.md:30]: Yes, we can add an entry saying: Provided documentation and workflows for replicating firsts Pixar's tutorials.

File: docs/integration_resources/software/resolve.md [hunk 8]
```diff
@@ -17,13 +17,13 @@ Apart from the official requirements, the `PYTHONHOME` environment is required t
 
 To use the Python interpreter provided with Shift, set up the variable in this way:
 
-**PYTHONHOME** : <path_to_your_shift_folder>/shift/thirdparty/python
+**PYTHONHOME** : <path_to_your_shift_folder&gt/shift/thirdparty/python
```
Comment (hectorjimenezinbibo) [inline docs/integration_resources/software/resolve.md:20]: With the change, on my side I have this result:
image

**Review by albertochimentiinbibo — APPROVED** [review]

File: docs/integration_resources/software/resolve.md (hunk 8 above)
Comment (juliavisserinbibo) [reply docs/integration_resources/software/resolve.md]: Done

File: docs/release_notes.md (hunk 6 above)
Comment (juliavisserinbibo) [reply docs/release_notes.md:36]: I added mentions to the examples

File: docs/release_notes.md (hunk 4 above)
Comment (hectorjimenezinbibo) [reply docs/release_notes.md:42]: Yes, after testing is confirmed that the latest distribution is working even in Houdini 20.5.332 with py310.

**Review by JorgeHernandezibanezinbibo — APPROVED** [review]

**Review by hectorjimenezinbibo — APPROVED** [review]

**Pull request merged**